tests/integration/integration_test.go

```diff
@@ -261,15 +260,22 @@ func loginUserWithPassword(t testing.TB, userName, password string) *TestSession
 // token has to be unique this counter take care of
 var tokenCounter int64
 
-func getTokenForLoggedInUser(t testing.TB, session *TestSession) string {
+// getTokenForLoggedInUser returns a token for a logged in user.
+// The scope is an optional list of snake_case strings like the frontend form fields,
+// but without the "scope_" prefix.
+func getTokenForLoggedInUser(t testing.TB, session *TestSession, scopes ...string) string {
 	t.Helper()
+	tokenCounter++
```

Comment:
This may result data race when running test parallel, use `atomic.AddInt64` instead.